src/test/java/org/broadinstitute/hellbender/tools/walkers/GenotypeGVCFsIntegrationTest.java

```diff
@@ -804,6 +805,9 @@ public void testWithReblockedGVCF() {
         final List<VariantContext> actualVC = VariantContextTestUtils.getVariantContexts(output);
         Assert.assertFalse(actualVC.stream().anyMatch(vc -> vc.getGenotype(1).isHomRef() && vc.getGenotype(1).hasPL()));  //second sample has a bunch of 0/0s -- shouldn't have PLs
 
+        //this comes from a callset of NYGC 1000G samples plus syndip
+        //it seems likely that there's a variant that wasn't discovered in the graph because a bunch of samples are hom-ref with PL=[0,0,X]
```

Comment:
a minor comment... but its possible to manufacture a sample with lots of assembly failures if you need to test this. You can set the kmer size to 1 and disallow stepped increases to get a sample that can assemble nothing.... Doesn't look necessary for you here though.

Reply:
Ooh, that is good to know.